lib/osf-components/addon/components/addons-service/user-addons-manager/component.ts

```diff
@@ -59,6 +59,14 @@ export default class UserAddonManagerComponent extends Component<Args> {
 
     @tracked selectedProvider?: Provider;
     @tracked selectedAccount?: AuthorizedStorageAccountModel;
+    @tracked credentialsObject = {
+        url: '',
+        username: '',
+        password: '',
+        token: '',
+        accessKey: '',
+        secretKey: '',
+    };
```

Comment:
Is `repo` missing from here?

Reply:
Oh, yep it is! It's stored on the `account-setup-manager` right now and that's not right